File: Opentrains_Contest/10316/B.cpp
#include <bits/stdc++.h>

using namespace std;

int n, m;

string s[1010], t, r;

int c[1010];

int main() {
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
	cin >> n;
	for (int p, i = 1; i <= n; i ++) {
		cin >> t, p = 0;
		r = t.substr(t.find('.') + 1);
		for (int j = 0; j < m; j ++)
			if (s[j] == r) {
				c[j] ++;
				p = 1;
				break;
			}
		if (p) continue;
		s[m] = r, c[m] = 1, m ++;
	}
	for (int i = 0; i < m; i ++)
		cout << s[i] << ": " << c[i] << '\n';
	return 0;
}

File: Opentrains_Contest/10316/B.py
f = open('input.txt', 'r')
w = open('output.txt', 'w')
n, t, c = 0, [], []
for i in range(int(f.readline())):
	s = f.readline()
	p = s.index('.')
	r = s[p + 1:]
	while r[-1] == '\n': r = r[:len(r) - 1]
	for j in range(n):
		if t[j] == r:
			c[j] += 1
			break
	else:
		t.append(r)
		c.append(1)
		n += 1
for i in range(n):
	w.write(t[i] + ': ' + str(c[i]) + '\n')